packages/buildpacks/test/commands/buildpacks/remove.test.ts

```diff
@@ -45,7 +46,7 @@ describe('buildpacks:remove', () => {
         expect(ctx.stdout).to.equal(
           `Buildpack removed.
 `)
-        expect(ctx.stderr).to.equal(' ›   Warning: The LANGUAGE_PACK_URL config var is still set and will be used \n ›   for the next release\n')
+        expect(unwrap(ctx.stderr)).to.equal('Warning: The LANGUAGE_PACK_URL config var is still set and will be used for the next release\n')
```

Comment:
I wish we had a better way of doing this in oclif. Like if we could do `.warning('The LANGUAGE_PATH...')` in fancy test

Reply:
Perhaps something to consider down the road